pages/common/gnmic.md

```diff
@@ -0,0 +1,25 @@
+# gnmic
+
+> A gNMI command-line client.
+> Manage gNMI network device configuration and view operational data.
+> More information: <https://gnmic.kmrd.dev>.
```

Comment:
We may want to draw attention that there are tldr pages for subcommands, but let's get a 👍 / 👎  some of the other tldr collaborators before proceeding with this change. Thoughts?

```suggestion
> Some subcommands such as `get`, `set`, and `subscribe` have their own usage documentation, accessible via `tldr gnmic-subcommand`.

> More information: <https://gnmic.kmrd.dev>.
```